platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/tipset/TipsetEventCreatorTests.java

```diff
@@ -152,9 +153,11 @@ private void validateNewEvent(
             final boolean slowNode) {
 
         final EventImpl selfParent = events.get(newEvent.getHashedData().getSelfParentHash());
-        final long selfParentGeneration = selfParent == null ? -1 : selfParent.getGeneration();
+        final long selfParentGeneration =
+                selfParent == null ? -EventConstants.GENERATION_UNDEFINED : selfParent.getGeneration();
```

Comment:
Did you intend to but a `-` sign in front of the constant?

Reply:
nope :-)

Reply:
find replace failure.